integration_tests/test_dbapi.py

```diff
@@ -87,22 +90,45 @@ def test_select_query_result_iteration_statement_params(presto_connection):
         params=(3,)  # expecting all the rows with id >= 3
     )
 
-def test_select_query_result_string_statement_params(presto_connection):
+def test_select_query_result_multi_type_statement_params(presto_connection):
     """
     Tests string parameters in prepared statements
     """
     cur = presto_connection.cursor()
+    expected_uuid = uuid.uuid1()
     # test a simple string parameter and
     # a parameter with a quote in it
     cur.execute(
-        "select ?",
-        params=("six'",),
+        "SELECT ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?",
+        params=(
+            "six'",
+            datetime(2020,1,1,0,0,0),
```

Comment:
nit: while we do not have a formal code style estabilished here, i personally would prefer a space after a comma

Reply:
updated, thanks!